src/index.ts

```diff
@@ -68,11 +67,19 @@ app.get(`/:id`, (request, response) => {
 app.post(`/`, (request, response) => {
     // Check if it is a valid url
     const { url } = request.body;
-    if (!url || !config.urlRegex.test(url)) {
+    
+    try {
+        url = new URL(string);
+    } catch (_) {
         response.status(400).send(`Please specify a (valid) URL.`);
         return;
     }
 
+    if (!url.protocol === "http:" || !url.protocol === "https:")  {
```

Comment:
I think protocols like FTP should work too, so we can remove this `if` statement.

Reply:
No. `javascript:void(0)` World be valid the, too. However you can still just add ftp to the if.